vips/vip-500/bscmainnet.ts

```diff
@@ -0,0 +1,218 @@
+          ],
+        ],
+      },
+      // Add Market
+      {
+        target: marketSpec.vToken.comptroller,
+        signature: "_supportMarket(address)",
+        params: [marketSpec.vToken.address],
+      },
+      {
+        target: marketSpec.vToken.comptroller,
+        signature: "_setMarketSupplyCaps(address[],uint256[])",
+        params: [[marketSpec.vToken.address], [marketSpec.riskParameters.supplyCap]],
+      },
+      {
+        target: marketSpec.vToken.comptroller,
+        signature: "_setMarketBorrowCaps(address[],uint256[])",
+        params: [[marketSpec.vToken.address], [marketSpec.riskParameters.borrowCap]],
+      },
+      {
+        target: marketSpec.vToken.address,
+        signature: "setAccessControlManager(address)",
+        params: [ACCESS_CONTROL_MANAGER],
+      },
+      {
+        target: marketSpec.vToken.address,
+        signature: "setProtocolShareReserve(address)",
+        params: [PROTOCOL_SHARE_RESERVE],
+      },
+      {
+        target: marketSpec.vToken.address,
+        signature: "setReduceReservesBlockDelta(uint256)",
+        params: [REDUCE_RESERVES_BLOCK_DELTA],
+      },
+      {
+        target: marketSpec.vToken.address,
+        signature: "_setReserveFactor(uint256)",
+        params: [marketSpec.riskParameters.reserveFactor],
+      },
+      {
+        target: marketSpec.initialSupply.vTokenReceiver,
+        signature: "withdrawTreasuryBEP20(address,uint256,address)",
+        params: [marketSpec.vToken.underlying.address, marketSpec.initialSupply.amount, NORMAL_TIMELOCK],
+      },
+      {
+        target: marketSpec.vToken.underlying.address,
+        signature: "approve(address,uint256)",
+        params: [marketSpec.vToken.address, marketSpec.initialSupply.amount],
+      },
+      {
+        target: marketSpec.vToken.address,
+        signature: "mintBehalf(address,uint256)",
+        params: [NORMAL_TIMELOCK, marketSpec.initialSupply.amount],
+      },
+      {
+        target: marketSpec.vToken.underlying.address,
+        signature: "approve(address,uint256)",
+        params: [marketSpec.vToken.address, 0],
+      },
+      {
+        target: marketSpec.vToken.address,
+        signature: "transfer(address,uint256)",
+        params: [ethers.constants.AddressZero, marketSpec.initialSupply.vTokensToBurn],
+      },
+      (() => {
+        const vTokensMinted = convertAmountToVTokens(marketSpec.initialSupply.amount, marketSpec.vToken.exchangeRate);
+        const vTokensRemaining = vTokensMinted.sub(marketSpec.initialSupply.vTokensToBurn);
+        return {
+          target: marketSpec.vToken.address,
+          signature: "transfer(address,uint256)",
+          params: [marketSpec.initialSupply.vTokenReceiver, vTokensRemaining],
+        };
+      })(),
+      {
+        target: marketSpec.vToken.comptroller,
+        signature: "_setActionsPaused(address[],uint8[],bool)",
+        params: [[marketSpec.vToken.address], [7], true],
+      },
```

Comment:
We cannot pause ENTER_MARKET, because the asset is borrowable, and the Comptroller "enters the assets" when they are borrowed for the first time. The Collateral Factor set to zero will avoid using this asset as collateral

Reply:
Removed the pause command on bscmainnet [f533877](https://github.com/VenusProtocol/vips/pull/548/commits/f533877454e7fa5b036919614cb3dc96acf9cb1e)
Added an addendum vip to unpause enter market in the bsctestnet [72bc6e0](https://github.com/VenusProtocol/vips/pull/548/commits/72bc6e0e801555dc2ed06191b920f88ae08263d0)